packages/shared/auth0-client/src/HttpAuth0Client.ts

```diff
@@ -210,6 +210,17 @@ export default class HttpAuth0Client implements Auth0Client {
                     ResponseStatus.PasswordTooWeak,
                     error
                   );
+                } else if (
+                  error.response.data?.message?.code === 136 &&
+                  error.responses.data?.message?.description.startsWith(
+                    'PIN is not valid : PIN is trivial'
+                  )
```

Comment:
Is there somewhere in the Sierra docs we can link to in a comment for (a) the code and (b) the Sierra definition of triviality?

Reply:
Would you believe me if I said there doesn't seem to be a Sierra definition of triviality? I can link to the [general error codes,](https://techdocs.iii.com/sierraapi/Content/zAppendix/errorHandling.htm?Highlight=error) which includes 136 but there doesn't seem to be a set section for the specific codes within that or of 'PIN is trivial'.